redash/cli/users.py

```diff
@@ -76,6 +70,31 @@ def password(email, password):
         print "User [%s] not found." % email
 
 
+@manager.option('email', help="The invitee's email")
+@manager.option('name', help="The invitee's full name")
+@manager.option('inviterEmail', help="The email of the inviter")
+@manager.option('--admin', dest='is_admin', action="store_true", default=False, help="set user as admin")
+@manager.option('--groups', dest='groups', default=None, help="Comma seperated list of groups (leave blank for default).")
+def invite(email, name, inviterEmail, groups, is_admin=False):
+    org = models.Organization.get_by_slug('default')
+    groups = build_groups(groups, is_admin)
+    try:
+        userFrom = models.User.get_by_email_and_org(inviterEmail, org)
+        user = models.User(org=org, name=name, email=email, groups=groups)
+
+        try:
+            user.save()
+        except IntegrityError as e:
+            if "email" in e.message:
+                print "Cannot invite. User already exists [%s]" % email
+                return
```

Comment:
Move the `invite_user` call and the message print to be inside the try clause, and then you can avoid the early return. Also please add an `else` clause in case the integrity error is for another reason.